examples/image-streams/image-streams-centos7.json

```diff
@@ -690,7 +690,7 @@
               "openshift.io/display-name": "MongoDB 2.4",
               "description": "Provides a MongoDB 2.4 database on CentOS 7. For more information about using this database image, including OpenShift considerations, see https://github.com/sclorg/mongodb-container/tree/master/2.4/README.md.",
               "iconClass": "icon-mongodb",
-              "tags": "mongodb",
+              "tags": "hidden,mongodb",
```

Comment:
we've always said the stateful set implementation is the right one, i don't see a good reason to keep the older example around, it's just an attracter for more issues.

and yeah, i'm aware of the name change though obviously i haven't mentally switched.